projects/ad7616-st/src/platform/stm32/main.c

```diff
@@ -59,7 +59,7 @@ struct no_os_uart_desc *uart_desc;
 int main()
 {
 	ad7616_ip.spi_param = &ad7616_spi_ip;
-	ad7616_spi_extra_ip.get_input_clock = HAL_RCC_GetPCLK1Freq;
+	ad7616_spi_extra_ip.get_input_clock = HAL_RCC_GetPCLK2Freq;
```

Comment:
this is regarding the whole commit "Switch to SPI1". can be squashed in the previous commit where the project is added.